cfme/services/requests.py

```diff
@@ -193,6 +193,16 @@ def remove_request(self, cancel=False):
         view = navigate_to(self, 'Details')
         view.toolbar.delete.click(handle_alert=not cancel)
 
+    @variable(alias="rest")
+    def remove_request_rest(self):
```

Comment:
This is not properly implemented for varmeth.

This is not the original implementation of `remove_request`, that already exists on L188.

This should be decorated with `@remove_request.variant('rest')`.

The original `remove_request` method at L188 should be decorated with `@variable(alias='rest')` to register the method with varmeth.

The function name itself is fine, can't have the names match.

Reply:
@mshriver thanks for the above comment... I added changes just one thing; L188 l.e. main method `remove_request` implemented with UI so better to decorate with variable only so it will act as the main method and default. Let me know your view.